### DISCUSSION TITLE
EMSCRIPTEN_BINDINGS(module), what does the module parameter do?

### DISCUSSION THREAD

**Discussion by frank-pian** [emscripten-core/emscripten#20202] [Q&A] [answered]
When I put all the embind code into one module or into different modules according to different classes, it doesn't make a difference.What are the best practices?

**Comment by sbc100** [comment, marked as answer]
It doesn't mean anything, it just gives the auto-generator constructor function a unique name.    You can't use the same name twice in the same file, but otherwise you can put whatever you want there that is valid in a C identifier.
